packages/notifications-controller/package.json

```diff
@@ -0,0 +1,83 @@
+{
+  "name": "@metamask/notifications-controller",
+  "version": "0.0.0",
+  "description": "Manages New MetaMask decentralized Notification system.",
+  "keywords": [
+    "MetaMask",
+    "Ethereum"
+  ],
+  "homepage": "https://github.com/MetaMask/core/tree/main/packages/notifications-controller#readme",
+  "bugs": {
+    "url": "https://github.com/MetaMask/core/issues"
+  },
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/MetaMask/core.git"
+  },
+  "license": "MIT",
+  "sideEffects": false,
+  "exports": {
+    ".": {
+      "import": "./dist/index.mjs",
+      "require": "./dist/index.js",
+      "types": "./dist/types/index.d.ts"
+    },
+    "./package.json": "./package.json"
+  },
+  "main": "./dist/index.js",
+  "types": "./dist/types/index.d.ts",
+  "files": [
+    "dist/"
+  ],
+  "scripts": {
+    "build": "tsup --config ../../tsup.config.ts --tsconfig ./tsconfig.build.json --clean",
+    "build:docs": "typedoc",
+    "changelog:update": "../../scripts/update-changelog.sh @metamask/notifications-controller",
+    "changelog:validate": "../../scripts/validate-changelog.sh @metamask/notifications-controller",
+    "publish:preview": "yarn npm publish --tag preview",
+    "test": "jest --reporters=jest-silent-reporter",
+    "test:clean": "jest --clearCache",
+    "test:verbose": "jest --verbose",
+    "test:watch": "jest --watch"
+  },
+  "dependencies": {
+    "@contentful/rich-text-html-renderer": "^16.5.2",
+    "@metamask/utils": "^8.3.0",
+    "@metamask/base-controller": "^5.0.2",
+    "@metamask/controller-utils": "^9.1.0",
+    "@metamask/keyring-controller": "^16.0.0",
+    "@metamask/profile-sync-controller": "^1.0.0",
+    "@metamask/push-platform-notifications-controller": "^1.0.0",
+    "contentful": "^10.11.3",
+    "deepmerge": "^4.2.2",
+    "loglevel": "^1.8.1",
+    "nock": "^13.3.1",
```

Comment:
This is already present in `devDependencies`.

Reply:
Fixed.